2-extra/3-groceries.js

```diff
@@ -27,14 +27,27 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+let allGroceries = Object.values(weeklyMealPlan);
+allGroceries.forEach((aDaysGroceries) => {
```

Comment:
Question: Could we use an object to check for repeating items instead of using the `includes` function? If so would there be an advantage to this other approach?

Reply:
Changes now completed with the use of Object key to store values. Added a counter of how many items of each type were added. This code is more efficient as it does not scan with includes for every loop.